botocore/endpoint.py

```diff
@@ -152,10 +152,10 @@ def prepare_request(self, request):
     def _calculate_ttl(
         self, response_received_timestamp, date_header, read_timeout
     ):
-        local_timestamp = datetime.datetime.utcnow()
+        local_timestamp = datetime.datetime.now(datetime.timezone.utc)
```

Comment:
As noted above, this can be simplified to avoid fundamentally changing `local_timestamp`:

```suggestion
        local_timestamp = datetime.datetime.now(datetime.timezone.utc).replace(tzinfo=None)
```